test/integration/test_simple.rb

```diff
@@ -83,9 +83,69 @@ def test_update_single_package
       # ...and notably no test for adder, which must be disabled from the option override from above
       assert_equal ['simple', 'using-adder'], exes
     }
+    update_with_option_off_and_build_with_uri_shorthand_syntax = -> {
+      prj.create_lists_from_default_template package: <<~PACK
+        CPMAddPackage(
+          URI gh:cpm-cmake/testpack-adder@1.0.0
+          OPTIONS "ADDER_BUILD_TESTS OFF"
+        )
+      PACK
+      assert_success prj.configure
+      assert_success prj.build
+
+      exe_dir = File.join(prj.bin_dir, 'bin')
+      assert File.directory? exe_dir
+
+      exes = Dir[exe_dir + '/**/*'].filter {
+        # on multi-configuration generators (like Visual Studio) the executables will be in bin/<Config>
+        # also filter-out other artifacts like .pdb or .dsym
+        !File.directory?(_1) && File.stat(_1).executable?
+      }.map {
+        # remove .exe extension if any (there will be one on Windows)
+        File.basename(_1, '.exe')
+      }.sort
```

Comment:
Not very important, but this code is now duplicated three times in the test case. Could it be more readable if we turn it into a method?